ckanext/xloader/loader.py

```diff
@@ -37,7 +37,7 @@ def get_write_engine():
 MAX_COLUMN_LENGTH = 63
 
 
-def load_csv(csv_filepath, resource_id, mimetype='text/csv', logger=None):
+def load_csv(csv_filepath, resource_id, mimetype='text/csv', logger=None, unicode_headers=None):
```

Comment:
It looks like `unicode_headers` is a parameter only used for test purposes, which is something I'd really prefer to avoid. How about the test instead uses `helpers.change_config` to achieve the same thing, and drop this param? https://docs.ckan.org/en/latest/extensions/testing-extensions.html

Reply:
yes, that's exactly what I was looking for, I'll change it